src/app/portlets/dot-form-builder/resolvers/dot-form-resolver.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Observable, of } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { Resolve, ActivatedRouteSnapshot } from '@angular/router';
+import { take, map, switchMap } from 'rxjs/operators';
+import { DotLicenseService } from '@services/dot-license/dot-license.service';
+import { DotMessageService } from '@services/dot-messages-service';
+
+export interface DotUnlicensedPortlet {
+    title: string;
+    description: string;
+    links: {
+        text: string;
+        link: string;
+    }[];
+}
+
+/**
+ * Check is there is license to view the form portlet
+ *
+ * @export
+ * @class DotFormResolver
+ * @implements {(Resolve<DotCMSPortletFallback>)}
+ */
+@Injectable()
+export class DotFormResolver implements Resolve<DotUnlicensedPortlet> {
+    constructor(
+        private dotLicenseService: DotLicenseService,
+        private dotMessageService: DotMessageService
+    ) {}
+
+    resolve(_route: ActivatedRouteSnapshot): Observable<DotUnlicensedPortlet> {
+        return this.dotLicenseService.isEnterprise().pipe(
+            switchMap((isEnterprise: boolean) =>
+                isEnterprise ? of(null) : this.getUnlicensePortlet()
+            ),
+            take(1)
+        );
+    }
+
+    private getUnlicensePortlet(): Observable<DotUnlicensedPortlet> {
+        return this.dotMessageService
+            .getMessages([
+                'Forms-and-Form-Builder',
+                'Forms-and-Form-Builder-in-Enterprise',
+                'Learn-more-about-dotCMS-Enterprise',
+                'Contact-Us-for-more-Information',
+                'Still-use-legacy-form-builder'
+            ])
+            .pipe(
+                map((messages: { [key: string]: string }) => {
+                    return {
```

Comment:
aren't you missing the `image path` also?

Reply:
![image](https://user-images.githubusercontent.com/37185433/64463813-9bf0e780-d0c2-11e9-9d84-061bdb086bd4.png)

Reply:
Images out, added an icon.

Reply:
No image now, added an icon